src/pagecontent/specification__behaviors__persisting_application_state.md

```diff
@@ -1,8 +1,37 @@
+
 ### Requesting User Identity
+To persist application state, the DTR application will need to know who is currently using the application. SMART on FHIR provides methods for the application to obtain this information.
+
 #### SMART on FHIR App Launch Framework IG
-##### fhirUser and openid scopes
+Using the SMART on FHIR App Launch Framework IG, the DTR application should request [scopes for requesting identity data](http://hl7.org/fhir/smart-app-launch/scopes-and-launch-context/index.html#scopes-for-requesting-identity-data), namely `openid` and `fhirUser`. The DTR application can then retrieve the FHIR resource representing the current person and extract whatever identifiers it deems necessary for the persistance of application state.
+
 ### Usage Sessions
-#### Combination of user and patient
-#### Visibility of sessions across users at a provider
+The DTR application is initially launched in response to a Coverage Requirements Discovery request. This request is for a given set of orders for a particular patient.
+
+The DTR application usage session binds this set of orders and patient to a particular user. When persisting application state, it is the responsibility of the DTR application to record the relationship between the orders, patient and user.
+
+The DTR application application may be launched outside the context of a CRD request, at which point, it must be able to restore a usage session. This will be covered in detail in Section 4.4.8 - Launch outside of CRD
+
+#### Visibility of Usage Sessions to Other Users
+Depending on workflows within an organization, it may be desirable for one user to start a usage session, but for it to be completed by another user. Rules for which users of a DTR application should be authorized to see information on a particular set of patients are outside the scope of this implementation guide.
+
+The SMART on FHIR App Launch Framework IG and FHIR specification provide methods for discovering information about the user and how that user fits into the provider organization. With this information, the DTR application, possibly consulting with the Payer IT system, can decide which usage sessions to expose to a given user.
+
+As an example, the DTR application may retrieve a Practitioner resource by retrieving the URL supplied in the `fhirUser` claim. The DTR application may then query the CareTeam or PractitionerRole resources to determine their relationship to other staff in the provider organization and determine if their usage sessions should also be made available.
+
 ### Session Expiration
+While a user may need to suspend interaction with the DTR application, there may be a limit on the amount of time that a set of documentation templates and rules is valid. It is unreasonable to resume the DTR application for an order that was proposed five years in the past.
```

Comment:
Current:
It is unreasonable to resume the DTR application for an order that was proposed five years in the past.

Proposed:
For example, it is unreasonable to resume the DTR application for an order that was started five years in the past.

Reply:
I've accepted your wording change.